js/open-photo.js

```diff
@@ -0,0 +1,59 @@
+import { photos } from './mock.js';
+import { isEscapeKey, findElement } from './util.js';
+
+const picturesBlock = document.querySelector('.pictures');
+const bigPicture = document.querySelector('.big-picture');
+const closeButton = document.querySelector('.big-picture__cancel');
+const commentsBlock = document.querySelector('.social__comments');
+const commentTemplate = document.querySelector('#comment').content.querySelector('.social__comment');
+
+const closePhoto = () => {
+	bigPicture.classList.add('hidden');
+	document.body.classList.remove('modal-open');
+	document.querySelector('.social__comments').textContent = '';
+};
```

Comment:
А вот тут стоит отвязать функцию
```suggestion
	document.querySelector('.social__comments').textContent = '';
	document.removeEventListener('keydown', buttonKeydownHandler);
};
```